scripts/update-screenshots.sh

```diff
@@ -82,9 +82,17 @@ else
     echo "SauceConnect tunnel detected already running, will attempt to use that."
 fi
 
+capabilities=$(python <<EOF
+import json
+capabilities = json.loads(open('$SELTEST_CAPABILITIES').read())
+capabilities['tunnel-identifier'] = '$TRAVIS_JOB_NUMBER'
```

Comment:
./update-screenshots.sh runs locally, so there won't be a TRAVIS_JOB_NUMBER--I think this file should remain the same.